### ISSUE TITLE
November 2023 Goals

### ISSUE THREAD

**Issue by sanya301** [hackforla/HomeUniteUs#631]
By 11/30/23

#### Final reviews:
- [ ] Coordinator Dashboard #501
- [ ] Guest Application #533 
- [ ] Finalize Sidebar + top navigation

#### New Items:
- [ ] Guest Profile #589
- [ ] Host Dashboard #508 
- [ ] Host Application #509 

#### In progress:

- [ ] Coordinator invite guests #405
- [ ] Guest Account Confirmation #440
- [ ] Guest Dashboard endpoints - needs data/infra structure solidifying
- [ ] API authentication (cognito + IAM)
- [ ] Coordinator dashboard

### PM:
- [ ] Create Guest, Host and Coordinator Sidebar stories
- [ ] Coordinator Application review page
- [ ] Coordinator User story for comments 
- [ ] Matching epic
- [x] Update [status doc](https://docs.google.com/document/d/1mksBNqE9hc-bAW49mdHQDImkk8f92YewtwENNpqvFHc/edit)
- [ ] Just MVP user flow

**Comment by sanya301** [comment]
Eng:
- Coordinator invite guests https://github.com/hackforla/HomeUniteUs/issues/405 + Guest Account Confirmation 
    - Almost done, some edge cases for tests
-  Guest Dashboard endpoints: needs data/infra structure solidifying
    - Split these up: tasks and ?
-  API authentication (cognito + IAM): break up into 3-4 smaller issues
     - submitted PR for the first issue, some configuration related bugs
-  Coordinator dashboard
     - Early version is in, state stuff waiting on design

**Comment by jasika-ai** [comment]
Eng:
- Coordinator invite guests https://github.com/hackforla/HomeUniteUs/issues/405 + Guest Account Confirmation 
    - Almost done, some edge cases for tests
-  Guest Dashboard endpoints: needs data/infra structure solidifying
    - Split these up: tasks and ?
-  Coordinator dashboard
     - Need to create a ticket and get someone started

**Comment by sanya301** [comment]
By 11/30/23

### Design:

#### Final reviews:
- [ ] Coordinator Dashboard #501
- [ ] Guest Application #533  - required, all are required besides the ones that start with if yes/no
- [ ] Finalize Sidebar + top navigation

#### New Items:
- [ ] Guest Profile #589
- [ ] Host Dashboard #508 
- [ ] Host Application #509 

### Eng:

#### In progress:

- [ ] Coordinator invite guests #405
- [ ] Guest Account Confirmation #440
- [ ] Guest Dashboard endpoints - needs data/infra structure solidifying
- [ ] API authentication (cognito + IAM)
- [ ] Coordinator dashboard

### PM:
- [ ] Create Guest, Host and Coordinator Sidebar stories
- [ ] Coordinator Application review page
- [ ] Coordinator User story for comments 
- [ ] Matching epic
- [ ] Update [status doc](https://docs.google.com/document/d/1mksBNqE9hc-bAW49mdHQDImkk8f92YewtwENNpqvFHc/edit)
- [ ] Just MVP user flow

**Comment by sanya301** [comment]
By 11/30/23

### Design:

#### Final reviews:
- [ ] Coordinator Dashboard #501 
- [ ] Guest Application #533 Some more questions for PM
- [ ] Finalize Sidebar + top navigation

#### New Items:
- [ ] Guest Profile #589
- [ ] Host Dashboard #508 
- [ ] Host Application #509 

### Eng:

#### In progress:

- [ ] Coordinator invite guests #405 - PRs out
- [ ] Guest Account Confirmation #440 - PRs out
- [ ] Guest Dashboard endpoints - split it into 3 issues, being reviewed
- [ ] API authentication (cognito + IAM)
- [ ] Coordinator dashboard - general layout is in place

### PM:
- [ ] Create Guest, Host and Coordinator Sidebar stories - working on this
- [ ] Coordinator Application review page
- [ ] Coordinator User story for comments 
- [ ] Matching epic
- [x] Update [status doc](https://docs.google.com/document/d/1mksBNqE9hc-bAW49mdHQDImkk8f92YewtwENNpqvFHc/edit)
- [ ] Just MVP user flow
